aws/table_aws_redshift_cluster.go

```diff
@@ -373,6 +380,39 @@ func getRedshiftCluster(ctx context.Context, d *plugin.QueryData, h *plugin.Hydr
 	return nil, nil
 }
 
+func getRedshiftLoggingDetails(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	// TODO Put me in helper function
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+
+	var name string
+	if h.Item != nil {
+		name = *h.Item.(*redshift.Cluster).ClusterIdentifier
+	} else {
+		name = d.KeyColumnQuals["cluster_identifier"].GetStringValue()
+	}
```

Comment:
Why do we need this block for a hydrate function?

Reply:
yes , we dont need to fetch it from cluster_identifier.